fastlane_core/lib/fastlane_core/analytics/analytic_ingester_client.rb

```diff
@@ -0,0 +1,9 @@
+module FastlaneCore
+  class AnalyticIngesterClient
+    def launched_event(session: nil, launch_context: nil)
```

Comment:
`event_launched_context` feels better to me, because of the `event_completed_context` comment  I just made 😄